src/main/scala/chiseltest/legacy/backends/vcs/VcsExecutive.scala

```diff
@@ -95,16 +95,29 @@ object VcsExecutive extends BackendExecutive {
     val moreVcsCFlags = compiledAnnotations
       .collectFirst { case VcsCFlags(flagSeq) => flagSeq }
       .getOrElse(Seq())
+    val coverageFlags = Seq((compiledAnnotations collect Map(
```

Comment:
Would something like the second code block in the first answer to this [StackOverflow post](https://stackoverflow.com/questions/4784051/scala-partition-collect-usage) be cleaner? Basically, can you (and similarly for the code below):
- instead of collect with a Map, can each be a case statement, and
- instead of putting everything in a Seq and then flattening it, can you use the output of the collect directly, which should be an iterable?

Also, with use of collect, which should drop irrelevant annotations instead of emitting empty strings, can you drop the `if (item.isEmpty)`?

Reply:
Map to case done.

For the flattening the process is Seq(collect -> flatten -> distinct -> string joined by "+") which is why we then have the if (item.isEmpty) on in map, effectively prepending "-cm" to the string in the Seq (type used for flag passing) if it exists.